README.md

```diff
@@ -1,83 +1,182 @@
+
+5. When the build has finished, check to make sure the blog renders correctly on the [staging site](https://staging-openlibertyio.mybluemix.net/blog/). 
+
+   This is the final check before the post is published live on the [production site](https://openliberty.io/blog/).
+
+   If there are any problems found on the [staging site](https://staging-openlibertyio.mybluemix.net/blog/), you must resolve them quickly or revert the PR.
+   
+   Make any changes in the author's branch, and push to both `draft` and `staging`.
+   
+6. To publish the post, create a PR from `staging` branch to `prod` branch and add @lauracowen (or other admin) as approver.
+
+7. When the PR is approved, merge it into `prod`.
+
+12. Rebuild the [production site from the IBM Cloud console](https://console.bluemix.net/devops/pipelines/fcc7c3e9-9c40-4a58-8a7f-09c08413ab7d?env_id=ibm:yp:us-south).
+
+    When the build has finished, check that the post looks as expected on [OpenLiberty.io/blog](https://openliberty.io/blog/).
+
+    If the post's file name uses a future date, the post will not display on the production site until that the production site is rebuilt on that day or later.
+
+13. When the post is published, and any changes you made are in all three branches (`draft`, `staging`, and `prod`), delete the author's branch.
+
+You've published a post!
+
+
+
+## Authors and Editors: updating a published post
+
+If a published post on OpenLiberty.io/blog contains an error or needs updating in any way, anyone can create a PR to get it fixed.
+
+1. As for creating a new post (see above), clone the `blogs` repo and create a new branch based on the `prod` branch. You will do all your work in this branch.
+
+2. Open the file in an editor (e.g. [VSCode with the Asciidoc plugin](https://marketplace.visualstudio.com/items?itemName=joaompinto.asciidoctor-vscode)) and make any changes needed.
+
+3. If the tags need correcting, update the [blogs_tags.json](./blog_tags.json) file. If you add new tags, make sure to add the blog post's slug to the beginning of the `posts` arrays (1-2 entries per line) for each tag.
+
+4. Create a PR from your branch to the `draft` branch, then run the [draft site build from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io) to check that the changes are fine on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
+
+   Make any changes in your branch then push to `draft` branch again rebuild.
+
+5. When the post is ready to publish, create a PR from your branch to `staging branch` (not from `draft` branch) and add @lauracowen as reviewer.
+
+6. When approved, @lauracowen (or other admin) will merge to `staging`, then run the [build for the staging site from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io) and check that it looks fine [staging site](https://staging-openlibertyio.mybluemix.net/blog/).
```

Comment:
... that it looks fine **on the** [staging site](https://staging-openlibertyio.mybluemix.net/blog/).